### PR TITLE
Drop activesupport dependency

### PR DESCRIPTION
**Pull request by bdunne**
Drop support for Ruby 1.9.3 and test against 2.3.1
Rails 5 isn't compatible with older rubies.
Rather than dropping support for older rubies just for #blank?, we can define #blank? if it isn't already defined.

### CODE REVIEW

**Comment by bdunne** [comment]
@Fryguy Please review

**Comment by bdunne** [comment]
@carbonin Please review.

File: lib/linux_admin/disk.rb [hunk 1]
```diff
@@ -13,23 +13,27 @@ class Disk
...
       case unit
       when 'K' then
-        val.to_f.kilobytes
+        val.to_f * 1_024 # 1.kilobytes
       when 'M' then
-        val.to_f.megabytes
+        val.to_f * 1_048_576 # 1.megabyte
       when 'G' then
-        val.to_f.gigabytes
+        val.to_f * 1_073_741_824 # 1.gigabytes
```
Comment (carbonin) [inline lib/linux_admin/disk.rb]: So, *technically* a kilobyte is 1000 bytes right? https://en.wikipedia.org/wiki/Kilobyte has a decent chart, but which form is being used is usually ambiguous. I'm assuming you checked out that these were what `1.kilobytes` and such actually resolved to?

Comment (Fryguy) [inline lib/linux_admin/disk.rb]:
```ruby
[1] pry(main)> 1.gigabytes
=> 1073741824
```

File: lib/linux_admin/registration_system/subscription_manager.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def organizations(options)
       params.merge!(proxy_params(options))
 
       result = run!(cmd, :params => params)
-      parse_output(result.output).index_by {|i| i[:name]}
+      parse_output(result.output).each_with_object({}) { |i, h| h[i[:name]] = i }
```
Comment (Fryguy) [inline lib/linux_admin/registration_system/subscription_manager.rb]: Was about to suggest using `.index_by`...LOL...I would have failed that question if it came up on http://railshurts.com/quiz/

**Comment by carbonin** [comment]
LGTM :+1:

File: lib/linux_admin/disk.rb (hunk 1 above)
Comment (bdunne) [inline lib/linux_admin/disk.rb]: Yes, I used the value of `1.gigabyte`, `1.megabyte`, `1.kilobyte`, etc.

**Pull request merged**